recipes/z3/all/conanfile.py

```diff
@@ -70,39 +83,76 @@ def _configure_cmake(self):
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "Visual Studio": "15.7",
+            "clang": "5",
+            "apple-clang": "10",
+        }
+
+    def validate(self):
+        if tools.Version(self.version) >= tools.Version("4.8.11"):
```

Comment:
```suggestion
        if Version(self.version) >= "4.8.11":
```
from conan.tools.scm import Version

Also, do not need to convert both, only the left hand.